DiscordBot.Data/Models/DiscordUserDto.cs

```diff
@@ -1,5 +1,5 @@
 ﻿namespace DiscordBot.Models;
-
+#nullable disable
```

Comment:
Wouldn't be better to add null!; for each property instead. For example: 
public string Id { get; set; } = null!;

Reply:
Wouldn't declaring the types as nullable be better? So instead of:
```csharp
public string Id { get; set; } = null!;
```
Where you're fighting the compiler to force null into a non null value, you just do:
```csharp
public string? Id { get; set; };
```
to properly tell the compiler that the value can be null instead?

Reply:
> Wouldn't be better to add null!; for each property instead. For example: public string Id { get; set; } = null!;

Wouldn't declaring the types as nullable be better? So instead of:
```csharp
public string Id { get; set; } = null!;
```
Where you're fighting the compiler to force null into a non null value, you just do:
```csharp
public string? Id { get; set; };
```
to properly tell the compiler that the value can be null instead?